saleor/payment/__init__.py

```diff
@@ -74,22 +74,27 @@ class TransactionKind:
 
 class ChargeStatus:
     """
-    - Charged: Funds were taken off the customer founding source, partly or
-               completely covering the payment amount.
     - Not charged: No funds were take off the customer founding source yet.
+    - Partially charged: Funds were taken off the customer founding source,
```

Comment:
Shouldn't this be `customer's funding source`?

Reply:
Thansk, will update, sounds better

Reply:
Oh damn I just noticed ```funding``` is misspelled. :crying_cat_face: